audio/audio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://assets-global.website-files.com/63c799a4fa41261ff1f92fd4/64213c94885ad9858a1c2c2b_01-favicon%20(1).png" rel="shortcut icon" type="image/x-icon"/>
+  <link href="https://assets-global.website-files.com/63c799a4fa41261ff1f92fd4/64213c9730d27c215422b7f9_01-favicon%20(2).png" rel="apple-touch-icon"/>
+  <title>BlendVision One Web Player Samples | Audio</title>
+  <style>
+    * {
+      box-sizing: border-box;
+    }
+    html, body {
+      margin: 0;
+      font-size: '16px';
+    }
+    .player { 
+      height: 100vh;
+    }
+    .on-device .player { 
+      height: calc(100vh - 50px);
```

Comment:
We may use `svh` or `dvh` for this case, for sample projects, support of legacy browsers is less important.